test/jasmine/tests/axes_test.js

```diff
@@ -8217,3 +8216,65 @@ describe('shift tests', function() {
         });
     });
 });
+describe('test tickmode calculator', function() {
+    beforeEach(function() {
+        gd = createGraphDiv();
+    });
+
+    afterEach(destroyGraphDiv);
+
+    function generateTickConfig(){
+      standardConfig = {tickmode: 'array', ticks: 'inside', ticklen: 1, showticklabels: false};
+
+      // Number of ticks will be random
```

Comment:
Let's use `Lib.pseudoRandom` as well as `Lib.seedPseudoRandom` as it applied by other jasmine tests.